scripts/move-contacts/index.ts

```diff
@@ -0,0 +1,34 @@
+import { queueManager } from '../../shared/queues';
+import { Config } from '../../src/config';
+import { ChtApi } from '../../src/lib/cht-api';
+import ChtSession from '../../src/lib/cht-session';
+import MoveLib from '../../src/lib/move';
+import SessionCache from '../../src/services/session-cache';
+
+import allPlacesToMove from './laikipia-west.json';
+
+const username = 'kenn_sippell_medic_user_manager';
+const password = '';
+const authInfo = Config.getAuthenticationInfo('laikipia.echis.go.ke');
+const contactType = Config.getContactType('d_community_health_volunteer_area');
```

Comment:
This is a very neat solution especially given the re-use of components from he user management tool. We should definitely look at having an experience built out for this and make it accessible to a few user managers and learn with them.

Nice work @kennsippell

Reply:
and @paulpascal for the foundational work

Reply:
I think cht-user-management is made in a way such that it is a powerful scripting library for custom user management work.